File: generator/output/ts-model/instance-type.model.ts
class InstanceTypeModel {
  public id: number = 0;

  // deleted 是否删除
  public deleted: boolean = false;

  // name 名称
  public name: string;

  // type 规格
  public type: string;

  // price 价格（单位元）/小时
  public price: number = 0;

  public created_at: string;

  public updated_at: string;
}

const InstanceTypeModelFields = {
  id: "id",
  deleted: "deleted",
  name: "name",
  type: "type",
  price: "price",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {InstanceTypeModel, InstanceTypeModelFields};

File: generator/output/entityModel/InstanceTypeModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.math.BigDecimal;
import java.lang.Boolean;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* InstanceType
*/
@Data()
public class InstanceTypeModel {
  public Long id = null;

  /**
  * 是否删除
  */
  public Boolean deleted = null;

  /**
  * 名称
  */
  public String name = null;

  /**
  * 规格
  */
  public String type = null;

  /**
  * 价格（单位元）/小时
  */
  public BigDecimal price = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
